assets/section-multicolumn.css

```diff
@@ -36,13 +36,15 @@
   margin-right: auto;
 }
 
-.multicolumn .button {
+.multicolumn .button,
+.multicolumn .link {
   margin-top: 1.5rem;
 }
 
 @media screen and (min-width: 750px) {
-  .multicolumn .button {
-    margin-top: 4rem;
+  .multicolumn .button,
+  .multicolumn .link {
+    margin-top: 3rem;
   }
```

Comment:
We're going to want to update multicolumn to have a slider at least on tablet to match the rest of the section. So this might need some tweaking in the future.